app/create_file.py

```diff
@@ -1 +1,42 @@
-# write your code here
+import sys
+import os
+import datetime
+
+
+def file_write(*filename: tuple[str]) -> None:
+    text = open(os.path.join(*filename), "a")
+    text.write(
+        str(datetime.datetime.now().strftime("%Y-%m-%d %H:%M%:%S")) + "\n"
```

Comment:
```suggestion
        datetime.datetime.now().strftime("%Y-%m-%d %H:%M%:%S") + "\n"
```
This convert is redundant